Home.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="./style.css">
+    <script src="./modal.js" defer></script>
+    <script src="./script.js" defer></script>
+    <title>Conference for Pair programming</title>
+</head>
+<body>
+    <div class="logo-top">
+        <ul class="logo-list">
+            <li><i class="fa fa-facebook"></i></li>
+            <li><i class="fa fa-twitter"></i></li>
+            <li><a class="top-list" href="#" >English</a></li>
+            <li><a class="top-list" href="#">My Page</a></li>
+            <li><a class="top-list" href="#">Logout</a></li>
+        </ul>
+    </div>
+    <header id="header">
+        <nav class="navigation">
+            <ul data-visible="false" class="item-navigation">
+                <li class="item-logo">
+                    <img class="logo-nav" src="./Image/background-image/cc-global.png" alt="cc-log" >
+                    <span class="logo-text-home">CC 2022 <br> Pair Programming</span>
+                    <span class="material-symbols-outlined close-menu">close</span>
+                </li>
+                <li class="link-navigation">
+                    <a class="link-nav" href="./Home.html">Home</a>
+                    <a class="link-nav" href="./About.html">About</a>
```

Comment:
- [ ] please make sure to add the other links so can the user navigation the sections using these links 

- Yours 

![image](https://user-images.githubusercontent.com/61790587/176554472-0c7213b1-f2d8-4f94-96c9-200cfa461746.png)

- Figma 

![image](https://user-images.githubusercontent.com/61790587/176554491-d731c450-7b1f-42c9-baa0-6417956e5213.png)